app/assets/stylesheets/v3/layout_components/side_nav.scss

```diff
@@ -31,61 +49,125 @@
     color: $white;
   }
 
+  &__list {
+    display: flex;
+    flex-grow: 1;
+    justify-content: space-between;
+    list-style-type: none;
+    margin: 0;
+    cursor: default;
+  }
+
   &__content {
-    padding: $base-padding;
+    display: flex;
   }
 
-  &__list {
+  &__content-container {
+    @extend .side-nav__content;
+    flex-grow: 1;
+    justify-content: space-evenly;
```

Comment:
https://caniuse.com/#search=space-evenly This doesn't work in IE. I wonder if there is a fallback/other solution we can use for IE?
![image](https://user-images.githubusercontent.com/1906696/88588069-2fe9db80-d014-11ea-8c4e-6586b0493cfb.png)